travel-agency/public/js/ajax-city-handler.js

```diff
@@ -0,0 +1,98 @@
+$.ajaxSetup({
+    headers: {
+        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+    }
+});
+$('form').submit(function(event){
+    event.preventDefault();
+})
+function createCity(){
+    var currentPage = new URLSearchParams(window.location.search).get('page') || 1;
+    var jsonData = formToJson($('#add-city-form'),currentPage);
+
+    $.ajax({
+        url: '/',
+        method: 'POST',
+        data: JSON.stringify(jsonData),
+        dataType: 'JSON',
+        contentType: 'application/json',
+        cache: false,
+        processData: false,
+        success:function(response)
+        {
+            document.forms["add-city-form"].reset();
+            $('#name-error').text('');
+            $('#cities-table').html(response.updatedCitiesTable)
+            $('#pagination-links').html(response.updatedPaginationLinks);
+        },
+        error: function(xhr) {
+            if (xhr.status === 422) {
```

Comment:
What if the HTTP status code is something different than `422`? As a user, I'd like to be informed that something went wrong, at least through a default error message.